### PR TITLE
Include 'update' state in oauth page

### LINKED ISSUE #710: Poor error message after allowing github integration
I allowed github access on [matrix-org/matrix-rust-sdk](https://github.com/matrix-org/matrix-rust-sdk) for the element.io GitHub Integration. Afterwards I saw this screen:

image

The error message comes from [here](https://github.com/matrix-org/matrix-hookshot/blob/c6f975b39045b3b6ef6b6c56e7781be56850500b/web/oauth.tsx#L48)

### PR DESCRIPTION
**Pull request by Half-Shot**
Fixes #710 

image

### CODE REVIEW

File: changelog.d/717.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Add information about GitHub App Installs in 'update' state on the oauth status page
```
Comment (AndrewFerr) [inline changelog.d/717.bugfix:1]: Nitpick: end the sentence with a period (and we can consider updating the CI changelog script to check for that).

File: web/oauth.tsx [hunk 2]
```diff
@@ -31,11 +31,13 @@ function RenderOAuth() {
                 <code>{errcode}</code> {error}
             </p>
         </>;
-    } else if (result === 'pending') {
+    // Pending / update are mostly the same thing. Pending means a new app install, update means updating the existing app install.
+    } else if (result === 'pending' || result === 'update') {
         return <>
-            <h1>Your connection to { serviceName } {oauthKind} is pending.</h1>
+            <h1>The connection to your { serviceName } {oauthKind} is pending.</h1>
```
Comment (AndrewFerr) [inline web/oauth.tsx:37]: Curious: why are there spaces around `serviceName` but not `oauthKind` or `result`?

**Review by AndrewFerr — APPROVED** [review]
Looks good!  Just have some minor comments

File: changelog.d/717.bugfix (hunk 1 above)
Comment (Half-Shot) [reply changelog.d/717.bugfix]: Done. And yeah we should!

**Pull request merged**